scipy/linalg/tests/test_solvers.py

```diff
@@ -137,38 +137,48 @@ class TestSolveDiscreteARE(TestCase):
     cases = [
         # Difficult from a numerical standpoint, again from Laub, A. J.
         # (http://dspace.mit.edu/bitstream/handle/1721.1/1301/R-0859-05666488.pdf)
-        (np.matrix([[4, 3], [-9.0/2.0, -7.0/2.0]]),
-         np.matrix([[1,], [-1,]]),
-         np.matrix([[9, 6], [6, 4]]),
-         np.matrix([[1,],])),
+        (np.array([[4, 3], [-9.0/2.0, -7.0/2.0]]),
+         np.array([[1,], [-1,]]),
+         np.array([[9, 6], [6, 4]]),
+         np.array([[1,],])),
         # Another example from Laub
-        (np.matrix([[0.9512, 0], [0, 0.9048]]),
-         np.matrix([[4.877, 4.877], [-1.1895, 3.569]]),
-         np.matrix([[0.005, 0],[0, 0.02]]),
-         np.matrix([[1.0/3.0, 0],[0, 3]])),
+        (np.array([[0.9512, 0], [0, 0.9048]]),
+         np.array([[4.877, 4.877], [-1.1895, 3.569]]),
+         np.array([[0.005, 0],[0, 0.02]]),
+         np.array([[1.0/3.0, 0],[0, 3]])),
         # Complex a; real b, q, r
-        (np.matrix([[2, 1-2j], [0, -3j]]),
-         np.matrix([[0,], [1,]]),
-         np.matrix([[1, 0], [0, 2]]),
-         np.matrix([[1,],])),
+        (np.array([[2, 1-2j], [0, -3j]]),
+         np.array([[0,], [1,]]),
+         np.array([[1, 0], [0, 2]]),
+         np.array([[1,],])),
         # Real a, q, r; complex b
-        (np.matrix([[2, 1], [0, -1]]),
-         np.matrix([[-2j,], [1j,]]),
-         np.matrix([[1, 0], [0, 2]]),
-         np.matrix([[1,],])),
-        # Real a, b; complex q, r
-        (np.matrix([[3, 1], [0, -1]]),
-         np.matrix([[1, 2], [1, 3]]),
-         np.matrix([[1, -3j], [1-1j, 2]]),
-         np.matrix([[-2j, 2], [1j, 3]])),
+        (np.array([[2, 1], [0, -1]]),
+         np.array([[-2j,], [1j,]]),
+         np.array([[1, 0], [0, 2]]),
+         np.array([[1,],])),
+        # Real a, b; complex q, r (corrected for hermitian Q)
```

Comment:
Safe to take out the "corrected for hermitian Q" bit--if someone can't see this diff they won't know what it means.